packages/lit-dev-content/site/tutorials/content/carousel/02.md

```diff
@@ -0,0 +1,68 @@
+The carousel is now showing its first child item but there's no way
+to change what's displayed. Let's address that by adding a `selected` property
+to control which item is shown.
+
+The property should be reactive and we'll make it an accessor
+so we can customize what happens when it's set. The total number of selectable
+items will be the `childElementCount`, and we'll add a bit of code to make
+the value wrap if it goes out of range. The `advancing` property will
+be use later when we do animation. Because we've made a custom property accessor,
+we need to call `requestUpdate` to make the element update. See the
+[docs](https://lit.dev/docs/components/properties/#accessors) for
+more details about creating accessors.
```

Comment:
Tried to re-org the commands to (1) explain concept, (2) add code, (3) explain code. This is tricky to get right, and perhaps it can be refined in the upcoming consistency phase.